cli/config_file.rs

```diff
@@ -530,6 +530,7 @@ pub struct ConfigFileJson {
   pub import_map: Option<String>,
   pub lint: Option<Value>,
   pub fmt: Option<Value>,
+  pub scripts: Option<Value>,
```

Comment:
I think we should name everything "task"/"tasks". It's shorter than scripts and pretty easy to switch to.

Reply:
Yes, good idea. I'll do that.

Reply:
Done